src/grokcore/startup/debug.py

```diff
@@ -153,28 +152,30 @@ def providedBy(self, obj=None):
             obj = self.ctx
         return list(zope.interface.providedBy(obj))
 
+
 def get_context_by_path(context, path):
     for name in (p for p in path.split(PATH_SEP) if p):
         context = context[name]
     return context
 
+
 def path_completer(self, event):
     """TAB path completer for `cdg` and `lsg` commands."""
     relpath = event.symbol
 
-    context = grokd.get_start_context(relpath)
+    context = grokd.get_start_context(relpath)  # noqa: F821 undefined name
```

Comment:
I can't say, whether it is currently tested well, but I see no point in this file, where the undefined name `grokd` should come from. Further down it gets instantiated explicitly. I would suggest that here too.